cmd/kpod/logout.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containers/image/pkg/docker/config"
+	"github.com/kubernetes-incubator/cri-o/libpod/common"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	logoutFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "override-path",
```

Comment:
Add -i
I am a lousy typest, Not crazy about options with "-" in them.
Perhaps lets call this "input,i" or "infile,i"

Reply:
fixed.